deploy/clowdapp.yaml

```diff
@@ -0,0 +1,201 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: sources
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: sources
+  spec:
+    envName: ${ENV_NAME}
+    deployments:
+    - name: api
+      minReplicas: ${{MIN_REPLICAS}}
+#      web: true # for compatibility with clowder 0.3.0
+      webServices:
+        public:
+          enabled: true # port 8000 is default in ClowdEnvironment
+      podSpec:
+        image: ${IMAGE}:${IMAGE_TAG}
+        initContainers:
+        - command:
+          - entrypoint
+          args:
+          - seed_database
+          inheritEnv: false
```

Comment:
Is there a reason we don't want to use this? 

I think we could do `inheritEnv: true` here, and just move `SOURCES_ENV` to the main `podSpec`. that way we don't have duplicated keys.

Reply:
we don't need all ENVs in init container and don't need SOURCES_ENV in the main container, so it's not duplicated.
This seems more clear to me, so is there a reason for `inheritEnv: true`?

Reply:
Yeah I'm feeling my review here wasn't in-depth enough, I also forgot that the DB settings are getting read in from clowder now not from the ENV vars. So there is no duplication. I apologize.